openpype/hosts/blender/api/plugin.py

```diff
@@ -86,6 +120,15 @@ def create_blender_context(active: Optional[bpy.types.Object] = None,
     raise Exception("Could not create a custom Blender context.")
 
 
+def get_instance_list():
```

Comment:
The function name should be `get_instances_list`. A list must be of "thing**s**"